apis/checkpoint/finalized_blocks/root.yaml

```
@@ -0,0 +1,57 @@
+get:
+  operationId: getFinalizedBlockRoot
+  summary: Get finalized block root
+  description: Retrieves hashTreeRoot of finalized BeaconBlock/BeaconBlockHeader. |
+    Responds with 'Block not found' if block at a slot is either unavailable or not yet finalized.
```

Probably worth being specific that the response will be a 404, "Respond with 'Block not found'" makes me think of a 200 OK response with the content 'Block not found'.

```suggestion
    Responds with 404 if block at a slot is either unavailable or not yet finalized.
```